Kubernetes/helmcharts/azure-policy-addon-arc-connected-clusters/values.yaml

```diff
@@ -5,8 +5,8 @@
 ## Microsoft Azure Policy Add-on in Kubernetes
 azurepolicy:
   image:
-    name: "emmcmill/azure-policy-kubernetes"
-    tag: "ccs-test"
+    name: "mcr.microsoft.com/azure-policy/policy-kubernetes-addon-prod"
```

Comment:
Is this the latest image to use for tomorrow's bug bash? For now, you can specify test image.